app/ids-counts/example-not-actionable.html

```diff
@@ -0,0 +1,35 @@
+<ids-layout-grid>
+  <ids-text font-size="12" type="h1">Counts: Disabled</ids-text>
+</ids-layout-grid>
+<ids-layout-grid cols="5" gap="md">
+  <ids-layout-grid-cell>
+    <ids-counts disabled="true">
```

Comment:
```suggestion
    <ids-counts disabled="true">
```

i think maybe disabled is the wrong word here. Its not disabled looking its just not linked. Maybe we can do this with just the presence of a `href` or not instead of making a disabled property as that would imply to me it looks disabled..

Reply:
I don't know why I thought the issue description explicitly mentioned "disabled" when, because it didn't. My mistake, I was actually thinking the same thing, the presence of an href is all you need.